CHANGELOG.md

```diff
@@ -24,7 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Deprecate InstrumentationLibraryInfo and Add InstrumentationScope
   ([#2583](https://github.com/open-telemetry/opentelemetry-python/pull/2583))
 
-## [1.11.0-0.30b0](https://github.com/open-telemetry/opentelemetry-python/releases/tag/v1.11.0-0.30b0) - 2022-04-18
+## Version 1.11.0/0.30b0 (2022-04-18)
```

Comment:
these links didn't seem to work anyways, so I took the liberty to simplify/align with what we are doing in the Java repos, but I can bring them back and update the release automation regexes accordingly if you'd like